### ISSUE TITLE
Output node shape: replace 'time' with -1

### ISSUE THREAD

**Issue by frankenjoe** [audeering/audonnx#17]
As the following simple example shows we currently only replace 'time' in the shape of input nodes, but should do the same with output nodes:

```python
import torch
import audonnx


class TorchModel(torch.nn.Module):

    def __init__(
        self,
    ):
        super().__init__()

    def forward(self, x: torch.Tensor):
        return x


torch_model = TorchModel()

input = torch.randn(1, 100, 4)  # batch, time, feature
output = torch_model(input)

onnx_path = 'model.onnx'
torch.onnx.export(
    torch_model,
    input,
    onnx_path,
    input_names=['input'],
    output_names=['output'],
    dynamic_axes={
        'input': {1: 'time'},
        'output': {1: 'time'},
    },
    opset_version=12,
)

audonnx.Model(onnx_path)
```
```
Input:
  output:
    shape: [1, -1, 4]
    dtype: tensor(float)
    transform: None
Output:
  output:
    shape: [1, time, 4]   # should be [1, -1, 4]
    dtype: tensor(float)
    labels: [output-0, output-1, output-2, output-3]
```

**Comment by hagenw** [comment]
I guess the goal is not to replace `time` with `-1`, but every axis marked as dynamic, correct?

**Comment by frankenjoe** [comment]
Well, if an axis is marked as dynamic, then the according dimension of the node is set to `time`. But it's more convenient to replace it with -1.

**Comment by hagenw** [comment]
> ```python
> 'input': {1: 'time'},
> ```

Ah, ok, I thought that you set the name to time with

```python
'input': {1: 'time'},
```
Not always obvious what happens in ONNX.

**Comment by frankenjoe** [comment]
Yes, here you have to set it to time, which is also the value that they assign to the node. But then you the dimension of the node is something like `[1, 'time', 10]`, which you cannot directly use to reshape an array. That's why we replace 'time' with -1 in the nodes of the model. I will create an PR, then it will become clear.
